frontend/src/app/testResults/viewResults/actionMenuModals/TestResultCorrectionModal.test.tsx

```diff
@@ -86,11 +85,11 @@ const renderModal = (
 describe("TestResultCorrectionModal", () => {
   let component: any;
 
-  beforeEach(() => {
+  /*beforeEach(() => {
```

Comment:
nit: any reason we are keeping this or ok to clean up? 🤔

Reply:
I have cleaned this up. Thanks for catching it!.